src/core/js/views/componentView.js

```diff
@@ -25,24 +25,18 @@ define([
         renderState: function() {
             if (!Handlebars.partials['state']) return;
 
-			// do not perform if component has .not-accessible class
+            // the preferred way to indicate that a state is not required
+            if (this.model.get('_disableAccessibilityState')) return;
+            // do not perform if component has .not-accessible class
             if (this.$el.is(".not-accessible")) return;
 			// do not perform if component has .no-state class
             if (this.$el.is(".no-state")) return;
 
-			//remove pre-exisiting states
-            var $previousState = this.$(".accessibility-state").remove();
+            var $previousState = this.$(".accessibility-state");
 
-            //render and append state partial
-            var $rendered = $(Handlebars.partials['state']( this.model.toJSON() ));
+            var $rendered = $(Handlebars.partials['state']( _.extend(this.model.toJSON(), {a11yConfig:Adapt.config.get('_accessibility')}) ));
```

Comment:
multiple lines would be good
```js
var template = Handlebars.partials['state'];
var data =  _.extend(this.model.toJSON(), {
    a11yConfig: Adapt.config.get('_accessibility')
};
var rendered = template(data);
```
i don't think we need ``rendered`` as a jquery object and dom nodes, as in both cases we can use the raw html

Reply:
Done